mailbox/store/src/main/java/org/apache/james/mailbox/store/event/DefaultDelegatingMailboxListener.java

```diff
@@ -86,12 +87,12 @@ public void removeGlobalListener(MailboxListener listener, MailboxSession sessio
 
     @Override
     public void event(Event event) {
-        Collection<MailboxListener> listenerSnapshot = registry.getLocalMailboxListeners(event.getMailboxPath());
+        Collection<MailboxListener> listenerSnapshot = registry.getLocalMailboxListeners(event.getMailboxId());
         if (event instanceof MailboxDeletion && listenerSnapshot.size() > 0) {
-            registry.deleteRegistryFor(event.getMailboxPath());
+            registry.deleteRegistryFor(event.getMailboxId());
         } else if (event instanceof MailboxRenamed && listenerSnapshot.size() > 0) {
```

Comment:
The handle rename is no more required, right?

Reply:
Yes, and I removed it on second commit